contrib/builder/rpm/armhf/centos-7/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM multiarch/centos:7.2.1511-armhfp-clean
```

Comment:
Since right now there is only this CentOS build on arm, it would be too early to abstract away the intersections into the `generate.sh` script. As an example the line `yum groupinstall -y "Development Tools"` without the `--skip-broken` is failing on the ARM CentOS. The same line would probably work on fedora.
Since right now there is only this CentOS build on arm, it would be too early to abstract away the intersections into the `generate.sh` script. As an example the line `yum groupinstall -y "Development Tools"` without the `--skip-broken` is failing on the ARM CentOS. The same line would probably work on fedora.